polynote-server/src/main/scala/polynote/server/Server.scala

```diff
@@ -48,6 +63,8 @@ trait Server extends IOApp with Http4sDsl[IO] with KernelLaunching {
       case req @ GET -> "notebook" /: path :? DownloadMatcher(Some("true")) =>
         IO(logger.info(s"Download request for ${req.pathInfo} from ${req.remoteAddr}")) *> downloadFile(path.toList.mkString("/"), req, config)
       case req @ GET -> "notebook" /: _ => serveFile(indexFile, req, watchUI)
+      case req @ GET -> Root / "logs" => showDir(config.storage.logs, req)
+      case req @ GET -> Root / "logs" / path => showFile(s"${config.storage.logs}/$path", req)
```

Comment:
As long as you tested this against stuff like `/logs/../../../../etc/passwd` 😛

Reply:
Err, I guess the notebook download thingy would have the same issue in that case. We should try to use both of those as attack vectors and see if it's reasonably OK.

Reply:
Yeah I can try it out. maybe only allow relative paths or something.